react-typescript.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    jest: true,
+  },
+  extends: 'airbnb',
+  plugins: ['react'],
+  rules: {
+    curly: ['error', 'multi'],
+    indent: ['error', 2],
+    'jsx-a11y/label-has-associated-control': ['error', { assert: 'either', depth: 5 }],
+    'no-param-reassign': 'off',
```

Comment:
Why is this off?

https://eslint.org/docs/rules/no-param-reassign

The explanation to avoid param reassign makes a lot of sense to me